core/templates/dev/head/domain/feedback_message/FeedbackMessageSummaryObjectFactory.ts

```diff
@@ -17,12 +17,23 @@
    message domain objects.
  */
 
-var oppia = require('AppInit.ts').module;
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
 
-oppia.factory('FeedbackMessageSummaryObjectFactory', [function() {
-  var FeedbackMessageSummary = function(
-      messageId, text, updatedStatus, suggestionHtml, currentContentHtml,
-      description, authorUsername, authorPictureDataUrl, createdOn) {
+export class FeedbackMessageSummary {
+  messageId: number;
+  text: string;
+  updatedStatus: any;
+  suggestionHtml: any;
```

Comment:
These were not present in the corresponding spec files so I couldn't know what exactly their type was. 
Are you sure that this (and the others) are `string`?

Reply:
You'll have to backtrack and check this in the backend. (I think this will be a string)

Also, should we have a pattern to add a comment (explaining the reason) if the type is `any`?

Reply:
I have created #7165 to track this.

Reply:
Done!